src/caselawclient/xquery/move_judgment.xqy

```diff
@@ -0,0 +1,10 @@
+xquery version "1.0-ml";
+
+import module namespace dls = "http://marklogic.com/xdmp/dls" at "/MarkLogic/dls.xqy";
+
+declare variable $old_uri as xs:string external;
+declare variable $new_uri as xs:string external;
+
+let $judgment := fn:doc($old_uri)
+
+return dls:document-insert-and-manage($new_uri, fn:true(), $judgment)
```

Comment:
document-insert sounds like a create rather than a transform -- does this leave the document at the old URI?

Docs at https://docs.marklogic.com/dls:document-insert-and-manage feel like they agree that it's a new doc.